kernel/kernel-defaults/src/main/java/io/delta/kernel/defaults/internal/expressions/DefaultExpressionUtils.java

```diff
@@ -220,28 +220,6 @@ static void compareBinary(ColumnVector left, ColumnVector right, int[] result) {
         }
     }
 
-    /**
-     * Utility method to return the left child of the binary input expression
-     */
-    static Expression getLeft(Expression expression) {
-        List<Expression> children = expression.getChildren();
-        checkArgument(
-            children.size() == 2,
-            format("%s: expected two inputs, but got %s", expression, children.size()));
-        return children.get(0);
-    }
-
-    /**
-     * Utility method to return the right child of the binary input expression
-     */
-    static Expression getRight(Expression expression) {
-        List<Expression> children = expression.getChildren();
-        checkArgument(
-            children.size() == 2,
-            format("%s: expected two inputs, but got %s", expression, children.size()));
-        return children.get(1);
-    }
-
     static Expression childAt(Expression expression, int index) {
```

Comment:
why does this belong in `DefaultExpressionUtils` and not in `ExpressionUtils`?

Reply:
I can move it over. I left it since it's only used by the default module currently